hasher-matcher-actioner/hmalib/lambdas/api/api_root.py

```diff
@@ -85,6 +83,22 @@ def upload():
     }
 
 
+@app.get("/image/<key>")
+def image(key=None):
+    """
+    image endpoint
```

Comment:
Nit: comment seems a bit redundant. Could we either remove or explain what this endpoint is used for (matches page?)